### ISSUE TITLE
compile error when building --with-unixodbc (libfl.so: undefined reference to `yylex')

### ISSUE THREAD

**Issue by saultdon** [mdbtools/mdbtools#47]
Is this a bison/flex issue?

```
Linux archasus 3.12.13-1-ck #1 SMP PREEMPT Sat Feb 22 18:08:42 EST 2014 x86_64 GNU/Linux
bison (GNU Bison) 3.0.2
flex 2.5.38
unixodbc 2.3.2
```

I've tried compiling using --with-unixodbc=/usr/include, --with-unixodbc=/usr/lib and just --with-unixodbc but they result in same error.

`````` bash
Making all in odbc
make[2]: Entering directory '/tmp/makepkg/mdbtools-git/src/mdbtools/src/odbc'
  CC       odbc.lo
  CC       connectparams.lo
  CCLD     libmdbodbc.la
  CC       libmdbodbcW_la-odbc.lo
  CC       libmdbodbcW_la-connectparams.lo
  CCLD     libmdbodbcW.la
  CC       unittest.o
  CCLD     unittest
/usr/lib/gcc/x86_64-unknown-linux-gnu/4.8.2/../../../../lib/libfl.so: undefined reference to `yylex'
collect2: error: ld returned 1 exit status```
``````

**Comment by LukeShu** [comment]
The correct solution is to remove all references to `@LEXLIB@` from the Makefiles.

The references to `@LEXLIB@` have been wrong since ChangeLog entry `2001-09-29  Brian Bruns <camber@ais.org>` (which was imported from CVS to Git in ea8fd2a6ba6fe25095747a203cbfe803d1bb1563). Since then, it should not link against `-lfl`.  The Makefiles were linking it against `-lfl` anyway, which until now apparently didn't mess anything up.

**Comment by adamcrown** [comment]
I'm getting this same error on Arch Linux. Any updates on this?

**Comment by nirgal** [comment]
LukeShu: I don't understand. The Makefile's no longer are part of the project, you need to use autoreconf -i. They were generated files anyway. There no longer is any -lfl in the code. It's only added by autoreconf.

Can you elaborate?

**Comment by LukeShu** [comment]
Sorry I was unclear. By "Makefiles", I meant `src/*/Makefile.am`.

Here's how the Parabola distro handles it: https://projects.parabola.nu/abslibre.git/tree/pcr/mdbtools/PKGBUILD#n26

@adamcrown: Parabola is compatible with Arch, feel free to grab the binary or the PKGBUILD.  I'll go ahead and add it to AUR4, but there's a solid chance that I'll forget to keep it up to date.

**Comment by evanmiller** [comment]
Duplicate of #81, fix included in #162
